src/Symfony/Component/Form/Util/StringUtil.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Util;
+
+/**
+ * @author Issei Murasawa <issei.m7@gmail.com>
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+class StringUtil
+{
+    /**
+     * This class should not be instantiated.
+     */
+    private function __construct()
```

Comment:
Actually I think private constructor should be after too, but some util classes' constructor is first. (e.g. classes in `Security/Core`, even `FormUtil`).